src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -310,6 +310,11 @@ using namespace Microsoft::Console::Interactivity;
             {
             case ApiLevel::Win32:
             {
+                // We don't need an "Default IME" window for ConPTY. That's the terminal's job.
+                // -1, aka DWORD_MAX, tells the function to disable it for the entire process.
+                // Must be called before creating any window.
+                ImmDisableIME(DWORD_MAX);

```

Comment:
This avoids spawning the entire IME machinery inside ConPTY.

Reply:
Oh, this answers my question about `imm32`.

For `sources`, can you instead link `ext-ms-win-imm-l1-1-0`? It is an extension APIset and can be delay-loaded.

That should be sufficient.
No need to change the link line for the vcxproj.